package.json

```diff
@@ -71,9 +69,13 @@
     "time-grunt": "1.2.1"
   },
   "scripts": {
-    "test": "grunt test; grunt lint",
-    "start": "node --harmony-arrow-functions index.js",
-    "startdev": "iojs --harmony-arrow-functions index.js --development -c"
+    "build": "cp -r lib dist && babel --out-dir=dist index.js 'lib/**/*.js'",
+    "test": "grunt test",
+    "start": "node dist/index.js",
+    "startdev": "babel-node index.js --development -c",
+    "lint": "grunt jshint",
```

Comment:
i dont see a grunt lint configured in the develop branch. can you all square that one away?